src/routes/duty-routes.js

```diff
@@ -0,0 +1,132 @@
+import { ObjectId } from "@fastify/mongodb";
+import { createDuty } from "../models/duty.js";
+import {
+	deleteDutySchema,
+	getDutyByIDSchema,
+	getDutyByQuerySchema,
+	patchDutySchema,
+	postDutySchema,
+	putConstraintsSchema,
+} from "../schemas/duty-schemas.js";
+
+export async function dutyRoutes(fastify) {
+	fastify.post("/", { schema: postDutySchema }, async (request, reply) => {
+		try {
+			const newDuty = createDuty(request.body);
+			await fastify.mongo.db.collection("duties").insertOne(newDuty);
+			fastify.log.info({ duty: newDuty }, "Duty created successfully");
```

Comment:
```suggestion
			req.log.info({ duty: newDuty }, "Duty created successfully");
```

big difference between the two, see why!